library/data/registry-entry-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/api/package-info.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * <h2>Registry Item Attributes</h2>
```

Comment:
I want to think about the name of the module's primary use case for a bit.

Attribute feels a bit weird since it describes a characteristic of an object. Sure this is the case with dispatched attributes where those could have functional characteristics. However with things like booleans, it isn't clearly a characteristic of an object but rather a piece of attached data.

Reply:
Hm. Maybe "Registry Data Attachment API"?
- `RegistryEntryAttribute` -> `RegistryDataKey`
- `RegistryEntryAttributeHolder` -> `RegistryDataMap`